scripts/push_docs_to_branch.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+RASA_FOLDER=rasa
+TESTS_FOLDER=tests
+DATA_FOLDER=data
+TMP_DOCS_FOLDER=tmp-documentation
+TODAY=`date "+%Y%m%d"`
+# we build new versions only for majors
+PATTERN_FOR_NEW_VERSION="^refs/tags/[0-9]+\\.0\\.0$"
+MASTER_REF=refs/heads/master
+
+[[ ! $GITHUB_REF =~ $PATTERN_FOR_NEW_VERSION ]] \
+&& [[ $GITHUB_REF != $MASTER_REF ]] \
+&& echo "Not on master or major version, skipping." \
+&& exit 0
+
+NEW_VERSION=
+if [ "$GITHUB_REF" != $MASTER_REF ]
+then
+    NEW_VERSION=${GITHUB_REF/refs\/tags\//}
+fi
+
+# clone the $DOCS_BRANCH in a temp directory
+git clone --depth=1 --branch=$DOCS_BRANCH git@github.com:$GITHUB_REPOSITORY.git $TMP_DOCS_FOLDER
+
+echo "Updating the docs..."
+cp -R $DOCS_FOLDER/* $TMP_DOCS_FOLDER/docs
+# we need these because they are imported by docs files
+cp -R $RASA_FOLDER/* $TMP_DOCS_FOLDER/$RASA_FOLDER
```

Comment:
is there a reason not to copy over everything? this seems a bit brittle as it will break without errors if someone includes a file that is not located in e.g. `data/`

Reply:
yeah I agree. At first I thought that we'd rather keep the `documentation` branch as "clean" as possible and not include everything (code, tests, scripts, etc...) but it seems that it will bring up more issues than solving them. I was on the fence, thanks for your input!